src/binary.ts

```diff
@@ -311,42 +310,39 @@ export class Binary extends BSONValue {
 export type UUIDExtended = {
   $uuid: string;
 };
+
 const UUID_BYTE_LENGTH = 16;
+const UUID_WITHOUT_DASHES = /^[0-F]{32}$/i;
+const UUID_WITH_DASHES = /^[0-F]{8}-[0-F]{4}-[0-F]{4}-[0-F]{4}-[0-F]{12}$/i;
 
 /**
  * A class representation of the BSON UUID type.
  * @public
  */
 export class UUID extends Binary {
-  static cacheHexString: boolean;
```

Comment:
Can this safely be removed?  It's a public property

Reply:
Good catch, added it back and filed follow ups, we can deprecate it in the subtask I made and remove in v6